backend/package.json

```
@@ -94,7 +95,16 @@
       "json",
       "ts"
     ],
-    "rootDir": "src",
+    "rootDir": "./",
+    "roots": [
+      "<rootDir>"
+    ],
+    "modulePaths": [
+      "<rootDir>"
+    ],
+    "moduleDirectories": [
+      "node_modules"
+    ],
```

```suggestion
    "rootDir": "src",
```

Essa parte não me parece precisar de alterações, só se o projeto não estiver rodando sem elas (o que acredito que não seja o caso)